integration/image/list_test.go

```diff
@@ -42,12 +42,12 @@ func TestImagesFilterMultiReference(t *testing.T) {
 	filter.Add("reference", repoTags[1])
 	filter.Add("reference", repoTags[2])
 	options := types.ImageListOptions{
-		All:     false,
 		Filters: filter,
 	}
 	images, err := client.ImageList(ctx, options)
 	assert.NilError(t, err)
 
+	assert.Equal(t, len(images), 1)
```

Comment:
```suggestion
	assert.Check(t, is.Len(images, 1))
```

Reply:
I do want `Equal` so that the test ends here, if the len is not 1, if it's 0 for example, the next line will panic